assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -39,6 +39,8 @@ import useCanvasKeys from './useCanvasKeys';
 
 const FramesPageArea = withOverlay(
   styled(PageArea).attrs({
+    className: 'container web-stories-content',
```

Comment:
What are these added for?

Reply:
Merged that from master (and prior to that I removed it when merging with older master), but now I see that we don't have it anymore - will remove it.